dev/com.ibm.ws.security.wim.core/resources/com/ibm/ws/security/wim/util/resources/WimUtilMessages.nlsprops

```diff
@@ -270,4 +270,36 @@ CANNOT_WRITE_TO_READ_ONLY_REPOSITORY.useraction=Ensure that a write operation is
\ No newline at end of file
+REQUIRED_IDENTIFIERS_MISSING.useraction=Ensure that the identifier property is specified for each entity passed in. If an identifier is not specified, add the property and specify either the uniqueName or the uniqueId.
+
+KRB5_LOGIN_FAILED_CACHE=CWIML4553E: Kerberos login failed using Kerberos principal {0} and Kerberos credential cache (ccache) {1}.
+KRB5_LOGIN_FAILED_CACHE.explanation=Either the specified Kerberos principal is invalid, or the Kerberos credential cache (ccache) is invalid or expired.
+KRB5_LOGIN_FAILED_CACHE.useraction=Ensure a valid Kerberos principal is specified and the Kerberos credential cache (ccache) has not expired.
+
+KRB5_LOGIN_FAILED_KEYTAB=CWIML4554E: Kerberos login failed using Kerberos principal {0} and Kerberos keytab {1}.
+KRB5_LOGIN_FAILED_KEYTAB.explanation=Either the specified Kerberos principal is invalid, or the Kerberos keytab is invalid.
+KRB5_LOGIN_FAILED_KEYTAB.useraction=Ensure a valid Kerberos principal is specified and a valid Kerberos keytab containing the Kerberos principal.
+		
+KRB5_LOGIN_FAILED_DEFAULT_CACHE=CWIML4555E: Kerberos login failed using Kerberos principal {0} and the default Kerberos credential cache (ccache).
+KRB5_LOGIN_FAILED_DEFAULT_CACHE.explanation=Either the specified Kerberos principal is invalid, or the default Kerberos credential cache (ccache) is invalid or expired.
+KRB5_LOGIN_FAILED_DEFAULT_CACHE.useraction=Ensure a valid Kerberos principal is specified and the default Kerberos credential cache (ccache) has not expired.
+
+KRB5_LOGIN_FAILED_DEFAULT_KEYTAB=CWIML4556E: Kerberos login failed using Kerberos principal {0} and the default Kerberos keytab.
+KRB5_LOGIN_FAILED_DEFAULT_KEYTAB.explanation=Either the specified Kerberos principal is invalid, or the default Kerberos keytab is invalid.
+KRB5_LOGIN_FAILED_DEFAULT_KEYTAB.useraction=Ensure a valid Kerberos principal is specified and the default Kerberos keytab is valid.
+
+KRB5_TICKETCACHE_USED=CWIML4557I: The {0} LdapRegistry is configured with a Kerberos ticket cache (ccache) filename {1} and keytab filename {2}. The Kerberos credential cache (ccache) is used for Kerberos bind authentication to LDAP server.
+KRB5_TICKETCACHE_USED.explanation=In order to use the keytab file, the keytab filename must be specified and the Kerberos credential cache (ccache) filename must not be specified.
+KRB5_TICKETCACHE_USED.useraction=No action is required. To avoid this message, remove the LdapRegistry ticketCache attribute or the Kerberos keytab attribute.
+
+INVALID_KRB5_PRINCIPAL=CWIML4558E: The {0} Kerberos principal name is incorrectly formatted, or the realm name is missing, or a default realm name cannot be found.
+INVALID_KRB5_PRINCIPAL.explanation=The Kerberos principal name cannot be null. The principal name must either include the realm name or a default realm name must be defined in Kerberos configuration file.
```

Comment:
Second sentence is confusing; suggested change to fix:
The principal name must either include the realm name
to
Either the principal name must include the realm name,